goony.go

```diff
@@ -77,3 +82,20 @@ func outputFile(filename string) files.File {
 
 	return file
 }
+
+func startContentStream(options config.Options, file files.File, linesChannel chan string) {
+	if options.RunEndless == true {
+		stop := make(chan bool, 1)
+		catchCtrlC(stop)
+		go files.StreamContentForever(file, linesChannel, stop)
+	} else {
+		go files.StreamContent(file, linesChannel)
```

Comment:
I was thinking about that actually.
You're right about that `StreamContent` shouldn't necessary be split into a "regular" and an endless version.
The current design is mostly because of testing purposes. If I "hide" the fact that the streaming can be stopped via a channel, I'm not sure how I can actually verify that it does what its supposed to.
Hence why it's currently set up and dispatched in `main`.

I'll have think. ;)